GOVERNANCE.md

```diff
@@ -0,0 +1,63 @@
+
+## Community
+
+The FOSSBilling community is made up of individuals, companies, or groups who use and/or contribute to the FOSSBilling software.
+
+Anybody can use the core FOSSBilling software within the terms of the [license](https://github.com/FOSSBilling/FOSSBilling/blob/main/LICENSE). There are not and never will be any fees for usage, and no special requirements.
+
+We really appreciate user contributions but if you just want to download, install, and go, then that is also fine. If you do want to be more involved in the community then [GitHub](https://github.com/FOSSBilling/FOSSBilling) and [Discord](https://fossbilling.org/discord) are good places to start. 
+
+## Steering Committee
+
+The steering committee is made up of community members and developers who make a sustained contribution to, and are recognised as being interested in, the long term health of the FOSSBilling project. 
+
+Each member of the steering committee also takes ownership of the project on GitHub. This ensures that in the event of any one person abandoning FOSSBilling or being unavailable for some reason the project does not stagnate or stop.
+
+At this moment the Steering Committee includes six members - identified here by their GitHub usernames:
+
+* [0x6b.dev](https://github.com/0x6b-dev)
+* [andpavlenko](https://github.com/andpavlenko)
+* [BelleNottelling](https://github.com/BelleNottelling)
+* [Evrifaessa](https://github.com/evrifaessa)
+* [HunterNyan](https://github.com/HunterNyan)
+* [jaapmarcus](https://github.com/jaapmarcus)
+
+Members of the steering committee may recommend and appoint new members from the community at any time. A nomination for a new member will result in a discussion and a vote by the existing committee members. 
+
+Members may be removed from the committee if they resign, or if they do not participate in the community or contribute to the project for a period of 6 months. 
+
+## Decision making process
+
+Major decisions about the future of FOSSBilling will be made after community discussion and by consensus of the steering committee. 
+
+We aim to resolve all issues and propsals through discussion, debate and collaboration. We are not saying that we will never resort to polls or votes but we will use them with care. Poll results will be used to help inform debate and consensus but they will not be taken as binding and definitive, and will not be the sole deciding factors.
+
+Ideologically it would be great to involve all community members in every decision, but in practical terms it is simply not possible if we want to be able to move forward at a reasonable speed.
+
+For the same reasons, to allow faster decision making and development, the project operates a policy of ‘lazy consensus' which allows for much faster decision making within the community and means that most decisions can be made without the need for a formal vote.
+
+Decision making normally passes through these steps:
+
+1. Proposal (which may be formal or informal)
```

Comment:
Fix numbering.

Reply:
The numbering is fine. You can put any numbers you like in an ordered list in Markdown and it will display them all correctly as 1. 2. 3.